### PR TITLE
Reports controller refactorization

### PR DESCRIPTION
**Pull request by jaleszek**

### CODE REVIEW

File: app/models/reports/admin_report.rb [hunk 1]
```diff
@@ -18,4 +19,31 @@ def build
       Reports::AllEntries.new(year).build
     end
   end
+
+  def as_pdf
+    pdf_klass = case id
+    when "feedbacks"
```
Comment (matthewford) [inline app/models/reports/admin_report.rb]: Indent `when` as deep as `case`.

File: app/models/reports/admin_report.rb [hunk 2]
```diff
@@ -18,4 +19,31 @@ def build
...
+      FeedbackPdfs::Base
+    when "case_summaries"
```
Comment (matthewford) [inline app/models/reports/admin_report.rb]: Indent `when` as deep as `case`.

File: spec/features/admin/form_answers/admin_download_all_case_summary_pdf_spec.rb [hunk 3]
```diff
@@ -21,7 +21,8 @@
     it "should be links to download feedbacks" do
       FormAnswer::AWARD_TYPE_FULL_NAMES.each do |award_type, value|
         expect(page).to have_link('Download',
-          href: download_case_summary_pdf_admin_reports_path(
+          href: admin_report_path(
```
Comment (matthewford) [inline spec/features/admin/form_answers/admin_download_all_case_summary_pdf_spec.rb]: Align the parameters of a method call if they span more than one line.

File: spec/features/admin/form_answers/admin_download_all_feedbacks_spec.rb [hunk 4]
```diff
@@ -21,7 +21,8 @@
     it "should be links to download feedbacks" do
       FormAnswer::AWARD_TYPE_FULL_NAMES.each do |award_type, value|
         expect(page).to have_link('Download',
-          href: download_feedbacks_pdf_admin_reports_path(
+          href: admin_report_path(
```
Comment (matthewford) [inline spec/features/admin/form_answers/admin_download_all_feedbacks_spec.rb]: Align the parameters of a method call if they span more than one line.

**Comment by RuslanHamidullin** [comment]
Looks good :+1:

**Pull request merged**